io/cmd.go

```diff
@@ -51,7 +52,14 @@ func RunCmd(config CmdConfig) error {
 	if err != nil {
 		return err
 	}
-	return cmd.Wait()
+	err = cmd.Wait()
+	// If the command fails to run or doesn't complete successfully *ExitError is returned.
+	// We want to avoid the current run to exit ass well, so we will return new error with the original error message.
```

Comment:
Readers of this comment may not understand why the current run will exit. They are not necessarily aware that this code is invoked from an application which uses code-gangsta, and even if they are, they probably won't make the connection.
I suggest changing:
// We want to avoid the current run to exit ass well, so we will return new error with the original error message.
to:
// We would like to return a regular error instead of ExitError, because some frameworks (such as codegangsta used by JFrog CLI) automatically exit wheh this error is returned.